lukefi/metsi/data/formats/ForestBuilder.py

```diff
@@ -429,11 +431,11 @@ def convert_stand_entry(self, estand: ET.Element) -> ForestStand:
         operations = smk_util.parse_stand_operations(estand, target_operations='past')
         stand = self.set_stand_operations(stand, operations) # RSD records 19, 20, 21, 23, 25, 26, 27, 28 and 31
         stand.natural_regeneration_feasibility = False # RSD record 22
-        stand.development_class = 0 # RSD record 24
-        stand.forestry_centre_id = -1 # RSD record 29
+        stand.development_class = smk_util.parse_development_class(0) # RSD record 24
+        stand.forestry_centre_id = None # RSD record 29
         stand.forest_management_category = smk_util.parse_forest_management_category(stand_basic_data.CuttingRestriction) or 1  # 30
-        stand.municipality_id = None or -1 # RSD record 32
-        stand.area_weight_factors = (1.0, 1.0)
+        stand.municipality_id = None # RSD record 32
+        stand.stems_per_ha_scaling_factors = (1.0, 1.0)
```

Comment:
This PR is overriding more recent changes. Is this intentional?

Reply:
My mistake. Fixed.